spec/system/placements/schools/mentors/add_an_existing_claims_mentor_spec.rb

```diff
@@ -0,0 +1,132 @@
+require "rails_helper"
+
+RSpec.describe "School user adds an existing claims mentor", service: :placements, type: :system do
+  before do
+    stub_valid_teaching_record_response
+  end
+
+  scenario do
+    given_the_mentor_and_school_exist
+    and_i_am_signed_in
+
+    when_i_am_on_the_mentors_index_page
+    and_i_click_on_add_mentor
+    then_i_see_the_find_mentor_page
+
+    when_i_enter_the_trn_and_date_of_birth_for_an_existing_claims_mentor
+    and_i_click_on_continue
+    then_i_see_the_confirm_mentor_details_page
+
+    when_i_click_on_change
+    then_i_see_the_find_mentor_form_with_the_trn_and_date_of_birth_prefilled_for_edna_krabappel
+
+    when_i_click_on_continue
+    then_i_see_the_confirm_mentor_details_page
+
+    when_i_click_on_confirm_and_add_mentor
+    then_i_see_a_success_message_for_edna_krapabbel
+    and_i_see_the_mentors_index_page_with_edna_krabappel_listed
+  end
+
+  private
+
+  def given_the_mentor_and_school_exist
+    @school = build(:school, :placements, name: "Springfield Elementary")
+    @claims_mentor = build(:claims_mentor, trn: "6666666", first_name: "Edna", last_name: "Krabappel")
+    create(:claims_mentor_membership, school: @school, mentor: @claims_mentor)
+  end
+
+  def and_i_am_signed_in
+    given_i_am_signed_in_as_a_placements_user(organisations: [@school])
+  end
+
+  def when_i_am_on_the_mentors_index_page
+    page.find(".app-primary-navigation__nav").click_on("Mentors")
```

Comment:
agreed! I've done this for the non-multitab specs and I've kept "visit" in multitab, because my understanding is that a new tab initialises a blank page.